content/raw/chat/online_users.md

```diff
@@ -26,19 +26,26 @@ It's important for end users to know which community members are online or offli
 
 ## Use Cases
 
-### Setting state as online
+### Setting state as always online
 
-1. User selects state as `online`
+1. User selects state as `always online`
```

Comment:
What is the purpose of having an indicator showing always online? Having no indicator vs always offline is the same right?

To me the status indicator purpose is to help other to understand my status and if online then that means that I am available but the always online won't do that and might cause frustration for people talking to someone always offline.

Why not keeping it simple, e.g
When the client is not online, then it is just showing offline.
When the client is online: you can either choose online or offline

Sorry if totally missed the point of the always offline.

Reply:
@alaibe the point of having an 'always online' option it to respect user's privacy.  So a user can choose to project whatever status they want to, e.g. a user may wish to present themselves to the world as either inactive or online even if they are not actually inactive or online.  

Obviously 'Set status automatically' probably the most useful setting for most people, and therefor should be the default.  There are some legitimate purposes why a user may not wish to have their status set automatically, for example a very security conscious person might worry about inadvertently revealing the rough timezone they are in via an automatically set status.  At the end of the day one of our values is to give users tools to protect their privacy, so a user shouldn't be forced to automatically revel their online status, and therefor we need to offer an alternative of allowing users to optionally select their inactive or online status manually.

In your message above you proposed letting a user choose to project an inactive state when they are in fact active.  If we are letting users do this, why shouldn't we also let users do the opposite e.g. set an online state when they might actually be inactive.

@alaibe does that answer your question?

Reply:
@John-44 thanks for clarifying the purpose of always online. I get the idea is to give the users full control of their availability status.

My only concern left is that we diverge from what everyone is used too. (I was confuse about it ;) ). I would assume that we have some neat design to present the various option and how they behave? I don't think that the wording would be enough to clarify our intent.

Thanks for explaining.

Reply:
I agree with your concern about potential user confusion, that's why we've ended up using "Always Online" (instead of just "Online") for this settings, and also "Set status automatically" for the automatic option (even though this is quite a long string of text to have in this context).

![2022-01-03 18_06_02-Kuba⎜Desktop – Figma](https://user-images.githubusercontent.com/22846356/147964221-5b957cb9-f561-489a-8ca8-0e9e4109edfb.png)

How to improve get even clearer without adding more clutter or complexity is a challenge.  We could (for example) add description text under each of these options, but then this mini-dialogue would become too large and users would have to stare at the description text even after they had learnt how it worked.  Or we could perhaps had a "?" that opens up another window with a longer description of how the different status's work, but most people wouldn't click on it so that would only ever help a minority of users.  Current thinking is to see how well this design works when we launch, and if we discover that this issue is in fact a problem then we will loop back and try a way to mitigate the issues.  But we aren't totally sure that this will be an issue (it might be an issue, but also it might not), and we couldn't find a way to better communicate how these status's work without introducing additional elements that themselves have an additional cost.